### PR TITLE
build: Latest devbox update

### PR DESCRIPTION
**Pull request by jimmidyson**
**What problem does this PR solve?**:

**Which issue(s) this PR fixes**:
Fixes #

**How Has This Been Tested?**:

**Special notes for your reviewer**:

### CODE REVIEW

**Review by mhrabovcin — APPROVED** [review]

File: charts/cluster-api-runtime-extensions-nutanix/templates/role.yaml [hunk 1]
```diff
@@ -49,17 +42,6 @@ rules:
       - addons.cluster.x-k8s.io
     resources:
       - clusterresourcesets
-    verbs:
```
Comment (faiq) [inline charts/cluster-api-runtime-extensions-nutanix/templates/role.yaml:52]: why is this changed?

**Review by faiq — APPROVED** [review]

File: charts/cluster-api-runtime-extensions-nutanix/templates/role.yaml (hunk 1 above)
Comment (jimmidyson) [reply charts/cluster-api-runtime-extensions-nutanix/templates/role.yaml:52]: Changes in controller-gen I guess - this is a generated file.
